doc/howtos/realsense.rst

```diff
@@ -0,0 +1,34 @@
+Intel® RealSense™ cameras in IoT Reference OS Kit
```

Comment:
I'm not sure how these symbols should be inserted in the text. I've seen "(r)" used in other docs and we used to use "|reg|" and "|trade|" in Ostro. @dbkinder , could provide some guidance here?